src/scripts/background.js

```diff
@@ -266,15 +266,27 @@ const extractTime = (start, end) => {
     }
 }
 
+const cleanIntervals = (intervals = []) => {
+    return intervals.filter(interval => {
+        const intervalStart = interval.split('-')[0];
+        const intervalEnd = interval.split('-')[1];
```

Comment:
You can use destructuring like `const [intervalStart, intervalEnd] = interval.split('-')`.